Base/usr/share/man/man7/Tips-and-Tricks.md

```diff
@@ -0,0 +1,60 @@
+![Welcome Icon](/res/icons/32x32/app-welcome.png)
+
+## Name
+Tips and Tricks
+
+## Description
+This is a list of useful tips and tricks to help you make the most out of SerenityOS.
+
+## General
+* When on the Desktop or in File Manager, start typing and the nearest match will be selected.
+* Bold text in context menus hints at the default behavior of a double-click.
+* Hold `Ctrl` to accelerate mouse wheel interaction with sliders and spin boxes.
+* Hold `Ctrl` while activating a menu item to prevent that menu from closing.
+* Many applications will open a compatible file if you drag-and-drop it into their window.
+* Default file and protocol associations can be changed in `~/.config/LaunchServer.ini`.
+
+### Window Management
+* Double-click a window's title bar to maximize it.
+* Click on a window's icon to open the context menu.
+* Double-click on the edge of an application's window to maximize it in that direction.
+* Middle-click on a window's maximize button to extend the window vertically (this can be undone in the same way).
+* Resizable windows can be dragged to all sides or corners of the screen, where they will automatically resize to fill half or one-quarter of the screen.
+* Right-click on the Workspace Picker applet and select 'Workspace Settings' to easily customize the number and layout of Workspaces (virtual desktops).
+
+### Fun
+* It can help to get a second pair of `$ Eyes` on a problem… or fifty: `$ Eyes -n 100`.
+
+## Applications
+
+### [Assistant](help://man/1/Applications/Assistant)
+* Assistant can help you to quickly find files and applications. Summon it with `Super+Space`.
```

Comment:
(“summon” is a word i really like but maybe this should be more of simple english—an “open” could be better here?)

Reply:
“Launch” or “Open” is probably the right word.
I liked “Summon” because it suggests Assistant is always available in the background ready to appear at a moments notice (though sounds a little occult).
I agree we should go with plain English rather than trying to be elegant.

Reply:
“bring up” is another thing that comes to my mind; but if we open dialog windows, then why not the assistant? “launch” is a synonym for “run” or “execute” for me—this would contradict your point about assistant being ready in the background (on the other hand, thatʼs a technicality which does not really matter for the end user 🤔).

(i believe that technically weʼre opening assistantʼs _window_. you could try to go around that if you want—pretty simple and very accurate).

Reply:
"Open" it is 🙂